README.md

```diff
@@ -1,2 +1,351 @@
-# year-cohort-projectName (eg: 22-jul-projectName)
-project template repo
+# Recipes API
+
+## API documentation
+
+### Return/Create user userId - story-2
+
+- **URL**
+
+  /users
+
+- **Method:**
+
+  `POST`
+
+- **URL Params**
+
+  none
+
+  **Example:**
+
+  `/users`
+
+## get user
+
+- **Success Response:**
+  - **Code:** 200
+  - **Content:**
+
+```json
+{
+  "message": "Successfully retrieved user",
+  "data": {
+    "userId": "1"
+  }
+}
+```
+
+- **Error Response:**
+
+- **Code:** 500 SERVER ERROR
+- **Content:** `{"message": "Unexpected error", "data": [] }`
+
+#### add user if not existing
+
+- **Success Response:**
+  - **Code:** 201
+  - **Content:**
+
+```json
+{
+  "message": "Successfully created user",
+  "data": {
+    "userId": "1"
+  }
+}
+```
+
+- **Error Response:**
+
+  - **Code:** 500 SERVER ERROR
+    **Content:** `{"message": "Unexpected error"}`
+
+## Return all recipes for a user - story3
+
+- **URL**
+
+  /users/:userId/recipes
+
+- **Method:**
+
+  `GET`
+
+- **URL Params**
+
+  none
+
+  **Example:**
+
+  `/users/1/recipes`
+
+- **Success Response:**
+- **Code:** 200
+- **Content:**
+
+```json
+{
+  "message": "Successfully retrieved all recipes",
+  "data": [
+    {
+      "id": 1,
+      "name": "baps",
+      "instructions": "how to make baps",
+      "duration": 3
+    },
+    {
+      "id": 2,
+      "name": "baps",
+      "instructions": "how to make baps",
+      "duration": 3
+    }
+  ]
+}
+```
+
+- **Error Response:**
+
+  - **Code:** 400 BAD REQUEST
+    **Content:** `{"message": "Invalid user id", "data": [] }`
+
+  - **Code:** 500 SERVER ERROR
+    **Content:** `{"message": "Unexpected error", "data": [] }`
+
+## Add a recipe for a user - Story-3
+
+- **URL**
+
+  /users/:userId/recipes
+
+- **Method:**
+
+  `POST`
+
+- **URL Params**
+
+  none
+
+  **Example:**
+
+  `/users/1/recipes`
+
+- **Success Response:**
+  - **Code:** 201
+  - **Content:**
+
+```json
+{
+  "message": "Successfully created recipe",
+  "data": {
+    "recipeId": 1,
+    "name": "baps",
+    "instructions": "how to make baps",
+    "duration": 3,
+    "prep_time": 1,
+    "cook_time": 2
```

Comment:
So after you add something, are you going to do another database query to get this data out for the response? Is this useful? Returning the id yes might be useful, but I question the other stuff because of how this data will be used - and it doesn't match how a single recipe is returned by its id either (no ingredients array), so trying to use it in your front end to go directly to a details page just won't work.